.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: PHPUnit Tests
+        strategy:
+            fail-fast: true
+            matrix:
+                php-versions: [ '7.2', '7.3', '7.4', '8.0', '8.1', 'nightly' ]
+                monolog-versions: [ '1.*', '2.*' ]
+                include:
+                    -   php-versions: '7.1'
+                    -   php-versions: '7.1'
+                        composer-flags: '--prefer-lowest'
+                    -   php-versions: '8.1'
+                        minimum-stability: 'dev'
+                    -   php-versions: 'nightly'
+                        minimum-stability: 'dev'
+        continue-on-error: ${{ matrix.php-versions > '8.0' }}
```

Comment:
Continuing after a step has failed looks weird to me. I suspect that you used this as an equivalent of the `allow_failures` feature of Travis, but this is absolutely not an equivalent feature (and I haven't found a case where it is the right solution anyway)

Reply:
Removed